assets/src/components/books/detail/BookDetail.test.jsx

```diff
@@ -34,7 +38,7 @@ describe('Book Detail', () => {
     const bookDetail = shallowBookDetail({ ...testDefaultProps });
     expect(
       bookDetail.find('.modal-book__details div.modal-book__title').text(),
-    ).toEqual('Test Driven Development');
+    ).toEqual('Test Driven Development<BookActionButton />');
```

Comment:
Not sure if is the right way to do it, just did it to fix the tests :|

Reply:
It's an issue with some of the older tests that still use Enzyme's shallow rendering. We should get rid of it to use react-testing-library's mount instead, but that will probably require some refactoring.